src/i18n/messages/ar.json

```diff
@@ -7,7 +7,7 @@
   "payment.messages.enrollment.code.product.info.quantity.updated.header": "لقد حدّثنا كميتك.",
   "payment.messages.enrollment.code.product.info.quantity.updated.body": "في سلتك {quantity, plural,\n   one {رمز تسجيل واحد}\n   two {رمزان للتسجيل}\n   few {# رموز تسجيل}\n   many {# رمز تسجيل}\n   other {# رمز تسجيل}\n} تقدّر تكلفتها بـ {price}, ستصلك عبر البريد الإلكتروني.",
   "payment.messages.transaction.declined.body": "م نستطع معالجة دفعتك، رجاءً تحقق من معلومات الدفع الخاص بك أو تقرب من مصرفك أو مؤسستك المالية من أجل المساعدة.",
-  "payment.messages.transaction.error.sku": "Your cart has changed since navigating to this page. Please reload the page and verify the product you are purchasing.",
+  "payment.messages.transaction.error.basket_changed": "Your cart has changed since navigating to this page. Please reload the page and verify the product you are purchasing.",
```

Comment:
I usually avoid changing the language `.json` files as they will be modified/triggered by the Transifex jobs, plus avoiding any Transifex mismatch errors from happening, just FYI. Since the ID has changed, they would have to be re-translated, and I am not sure how changing these files manually what would happen

Reply:
ugh okay. i'll revert these changes then